### ISSUE TITLE
Error when run demo.py

### ISSUE THREAD

**Issue by shipengai** [eragonruan/refinenet#14]
'''NotFoundError (see above for traceback): Key feature_fusion/Conv_37/BatchNorm/beta/ExponentialMovingAverage not found in checkpoint'''
The ckpt is your trained model.

Thanks.

**Comment by shipeng-uestc** [comment]
/Users/mac/Desktop/1111111111.jpg

**Comment by eragonruan** [comment]
@shipeng-uestc check your tensorflow version

**Comment by EricZgw** [comment]
NotFoundError (see above for traceback): Key feature_fusion/Conv_46/BatchNorm/gamma/ExponentialMovingAverage not found in checkpoint
I have the save problem,my tensorflow version is 1.4.1.

**Comment by Moyao6** [comment]
@eragonruan Could you give out your tensorflow version, I met this problem too.

**Comment by CabbageWust** [comment]
请问是哪个版本
